<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="10383" started-at="2019-04-09T21:11:31Z" finished-at="2019-04-09T21:11:41Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="10383" started-at="2019-04-09T21:11:31Z" finished-at="2019-04-09T21:11:41Z">
      <class name="com.automation.app.SignupTests">
        <test-method status="PASS" signature="beforeSuite(java.lang.String)[pri:0, instance:com.automation.app.SignupTests@42d80b78]" name="beforeSuite" is-config="true" duration-ms="11045" started-at="2019-04-09T21:11:20Z" finished-at="2019-04-09T21:11:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="reachSignupPage()[pri:0, instance:com.automation.app.SignupTests@42d80b78]" name="reachSignupPage" duration-ms="10351" started-at="2019-04-09T21:11:31Z" finished-at="2019-04-09T21:11:41Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.automation.app.SignupTests.reachSignupPage(SignupTests.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reachSignupPage -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.automation.app.SignupTests@42d80b78]" name="afterSuite" is-config="true" duration-ms="1329" started-at="2019-04-09T21:11:41Z" finished-at="2019-04-09T21:11:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.automation.app.SignupTests -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
